rust/core-lib/src/config.rs

```diff
@@ -328,6 +328,12 @@ impl ConfigManager {
     pub fn default_buffer_config(&self) -> BufferConfig {
         self.get_buffer_config(None, None)
     }
+
+    /// Path to themes sub directory inside config directory
+    pub(crate) fn get_themes_dir(&self) -> PathBuf {
+        let themes_dir = self.config_dir.as_ref().unwrap().join("themes");
```

Comment:
this will crash clients that don't use a config dir, of which several exist.

Reply:
Yeah, I'm working on it. Haven't pushed its fix yet.

Reply:
Fixed.